### PR TITLE
Introduce black.

### PR DESCRIPTION
**Pull request by hramezani**
## Solution

- Format codes with black
- Add `format` entry to `Makefile`
- Add `Code Formatting` section to `CONTRIBUTING.rst`
- Add `black` to tox envlist

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [ ] Update documentation (if relevant).

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -76,3 +76,13 @@ To run tox and generate a coverage report (in ``htmlcov`` directory)::
...
 **Please note**: Before a pull request can be merged, all tests must pass and
 code/branch coverage in tests must be 100%.
+
+Code Formatting
+---------------
+We make use of `black`_ for code formatting.
+
+.. _black: https://black.readthedocs.io/en/stable/installation_and_usage.html
+
+Once it is installed you can make sure the code is properly formatted by running::
```
Comment (smithdc1) [inline CONTRIBUTING.rst:86]: Could add via a pre-commit hook? I find it very helpful when a codebase has black (enivitably I forget to run it). :shrug:

File: tox.ini [hunk 2]
```diff
@@ -37,3 +38,11 @@ ignore =
...
     E402  ; module level import not at top of file
     E501  ; line too long
+
+[testenv:black]
+basepython =
+    python3.7
+deps =
+    black==20.8b1
+commands =
+    black --check model_utils tests
```
Comment (smithdc1) [inline tox.ini:48]: `setup.py` as well?

**Review by smithdc1 — COMMENTED** [review]
Out of scope here, but could add isort as well?

File: CONTRIBUTING.rst (hunk 1 above)
Comment (hramezani) [reply CONTRIBUTING.rst:86]: Yes, would be great I think :+1:

File: tox.ini (hunk 2 above)
Comment (hramezani) [reply tox.ini]: Yes, added!

**Comment by hramezani** [comment]
> Out of scope here, but could add isort as well?
Yes, would be great.

**Comment by hramezani** [comment]
I realized that we have [flake8 in tox](https://github.com/jazzband/django-model-utils/blob/882919f7e0bb2eeb7ce299702f50d7f17d81c705/tox.ini#L4) but we don't have it in travisci.
I am going to close this PR and open another one to include flake8 in our travis ci.

Thanks @smithdc1 for review.

**Pull request closed without merging**